tests/e2e/Services/Projects/ProjectsConsoleClientTest.php

```diff
@@ -429,6 +429,35 @@ public function testUpdateProject($data): array
         return ['projectId' => $projectId];
     }
 
+    /**
+     * @depends testCreateProject
+     */
+    public function testUpdateProjectSMTP($data): array
```

Comment:
Can we also test read?
I would also validate the logic to get the default server template if nothing has been set.

Reply:
updated